src/hello_world.cpp

```diff
@@ -144,27 +145,25 @@ NAN_METHOD(HelloWorld::shout)
 // this is where we actually exclaim our shout phrase
 void HelloWorld::AsyncShout(uv_work_t* req)
 {
-    helloworld_shout_baton *baton = static_cast<helloworld_shout_baton *>(req->data);
+    AsyncBaton *baton = static_cast<AsyncBaton *>(req->data);
 
     /***************** custom code here ******************/
-
-    std::string return_string = baton->phrase + "!";
-
-    if (baton->louder)
+    try
     {
-        return_string += "!!!!";
-    }
 
-    /***************** end custom code *******************/
+        std::string return_string = baton->phrase + "!";
 
-    try
-    {
-        baton->result = return_string;
+        if (baton->louder)
```

Comment:
@springmeyer seeing that the `return_string` now [comes back empty](https://travis-ci.org/mapbox/node-cpp-skel/jobs/138593983#L338) for this method after wrapping it all within the `try/catch`. Seems odd, but if I pull it out:

```
/***************** custom code here ******************/

std::string return_string = baton->phrase + "!";

if (baton->louder)
{
    return_string += "!!!!";
}

/***************** end custom code *******************/

try
{
    baton->result = return_string;
}
catch (std::exception const& ex)
{
    baton->error_name = ex.what();
}
```

things work fine